docker-compose.prod.yml

```
@@ -0,0 +1,75 @@
+oauth2-proxy:
+    container_name: temporal-oauth-proxy
+    image: quay.io/oauth2-proxy/oauth2-proxy:v7.8.0
+    restart: unless-stopped
+    depends_on:
+      - temporal-ui
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)